src/semVer.ts

```diff
@@ -51,13 +51,20 @@ export function semVerFromString(semVer: string): SemVer {
 		} else if (semVer[0] == '~') {
 			if(semVer[1] == '>'){
 				rubyGemsSkipLast = true
-				semVer = semVer.substring(2)
+				semVer = semVer.substring(1)
 			}
 			semVer = semVer.substring(1)
 			skipBug = true
 		} else if (semVer[0] == '=') {
 			semVer = semVer.substring(1)
-		} else{
+		} else if (semVer[0] == '>'){
+			if(semVer[1] == '>'){
+				semVer = semVer.substring(1)
+			}
+			semVer = semVer.substring(1)
+			skipMinor = true
+			skipBug = true
+		}else{
 			break
 		}
 	}
```

Comment:
**Not important.**

Do you guys think is possible to just use a regex to do the semver string parsing? It may considerably reduce the amount of code required to do this parsing, reduce complexity and improve maintainability.